src/utils/__tests__/warning.test.ts

```diff
@@ -0,0 +1,30 @@
+import {warning} from '../warning'
+
+test('emits a message to console.warn() when the condition is `false`', () => {
```

Comment:
Should this be 'emits a message to console.warn() when the condition is `true`' ?

`warning(true, 'test')` -> warn if condition true 🤔  Or did I confuse myself?

Reply:
You're right @broccolinisoup! Sorry about that, I updated the implementation but forgot to change the test title. I'll make that update now 👍